### PR TITLE
Fix/ola

### PR DESCRIPTION
**Pull request by euhmeuh**
MANAGER-3148

### CODE REVIEW

File: client/app/dedicated/server/interfaces/interfaces.routing.js [hunk 1]
```diff
@@ -18,6 +18,13 @@ export default /* @ngInject */ ($stateProvider) => {
       ) => (translateId, error) => $timeout(() => {
         Alerter.set('alert-danger', $translate.instant(translateId, { error: error.message }));
       }),
+      alertSuccess: /* @ngInject */ (
+        $timeout,
+        $translate,
+        Alerter,
+      ) => translateId => $timeout(() => {
```
Comment (marie-j) [inline client/app/dedicated/server/interfaces/interfaces.routing.js]:
```suggestion
      ) => message=> $timeout(() => {
```
would be better imo and consistent with the rest (or at least the bits that are)

File: client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js [hunk 2]
```diff
@@ -62,10 +64,11 @@ export default class {
     this.selectedInterfaces = selectedRows;
 
     if (this.configuration.mode === OLA_MODES.DEFAULT) {
-      this.networkInterfaces = _.flatten(_.map(
-        this.selectedInterfaces,
-        ({ mac }) => mac.split(','),
-      ));
+      const macs = this.selectedInterfaces[0].mac.split(',');
```
Comment (marie-j) [inline client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js]: We're sure it's defined ?

File: client/app/dedicated/server/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -58,6 +58,7 @@
   "server_tab_stats_network": "Réseau :",
   "server_tab_stats_network_public": "Interface réseau publique ({{t0}})",
   "server_tab_stats_network_private": "vRack ({{t0}})",
+  "server_tab_stats_network_private_lag": "Interface privée aggrégée ({{t0}})",
```
Comment (marie-j) [inline client/app/dedicated/server/translations/Messages_fr_FR.json]:
```suggestion
  "server_tab_stats_network_private_lag": "Interface privée aggrégée ({{interface}})",
```
I think it's better to explicit what the expected translate argument is

File: client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js [hunk 4]
```diff
@@ -62,10 +64,11 @@ export default class {
...
+      this.fakeNetworkInterfaces = [
```
[the remark this answers is not in the document]
Comment (euhmeuh) [reply client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.controller.js]: Yes, `simulated` is better!

File: client/app/dedicated/server/interfaces/interfaces.routing.js (hunk 1 above)
Comment (euhmeuh) [reply client/app/dedicated/server/interfaces/interfaces.routing.js]: The thing is, `message` is also used for the `error.message` attribute, which brings confusion. That's why I opted for the explicit name `translationId`.

**Comment by antleblanc** [comment]
Please refer to https://github.com/ovh/manager

**Pull request closed without merging**